### ISSUE TITLE
All ONCLICK handlers should have an equivalent ONKEYPRESS handler.

### ISSUE THREAD

**Issue by lfrancis** [cultureondemand/iava-report#4]
All ONCLICK handlers should have an equivalent ONKEYPRESS handler.
WCAG 2.0 A F54 Section 508 (2017) A F54
1 pages

Some users are unable to use a mouse, so use the keyboard instead. Add an equivalent keyboard event handler to help these users.

https://cultureondemand.github.io/iava-report/
Line 198
